src/main/webapp/i18n/de/complaint.json

```diff
@@ -46,6 +34,28 @@
                 "rejected": "Abgelehnt",
                 "showAddressedComplaints": "Adressierte Beschwerden anzeigen",
                 "noComplaintsToShow": "Es gibt im Moment keine Beschwerden zu zeigen."
+            },
+            "review": "Beschwerde überprüfen"
+        },
+        "moreFeedback": {
+            "title": "Weiteres Feedback anfragen",
+            "description": "Bitte den Tutor, dir weiteres Feedback zu geben, wenn etwas nicht klar genug ist oder du weitere Erklärungen benötigst.",
+            "beDescriptive": "Bitte beschreibe ausführlich, was für dich in der Bewertung unklar ist. Deine Anfrage wird dann von demselben Prüfer bewertet.",
```

Comment:
Sometimes you write `Prüfer`, sometimes you write `Tutor` --> This is inconsistent. I think even in German, we should write Reviewer, because this has become a normal word in German. Please also double check all other translations in German regarding this